docker-compose.yml

```diff
@@ -1,21 +1,11 @@
-# Use root/example as user/password credentials
 version: '3.1'
 
 services:
-
-  mongo:
-    image: mongo
+  db:
+    image: postgres
     restart: always
     environment:
-      MONGO_INITDB_ROOT_USERNAME: root
-      MONGO_INITDB_ROOT_PASSWORD: example
-
-  mongo-express:
-    image: mongo-express
-    restart: always
+      POSTGRES_PASSWORD: password  # Change this to your desired password
```

Comment:
Do we need to move password to secrets?

Reply:
ik heb geen idee hoe we ik met secrets zou moeten werken, is dit iets makkelijks? Ik stel voor om voor nu het even zo te houden.